pyproject.toml

```diff
@@ -1,19 +1,34 @@
 [tool.poetry]
 name = "fmugym"
-version = "0.1.0"
-description = ""
+version = "0.1.1"
```

Comment:
Updated version number. Once merged, create a Git(Hub) tag named v0.1.1 to have workflow triggered for PyPi publishing